muxers/mplex/src/config.rs

```diff
@@ -84,6 +88,23 @@ impl MplexConfig {
         self.split_send_size = size;
         self
     }
+
+    /// Gets the protocol anme
+    pub fn protocol_name(&self) -> &[u8] {
+        self.protocol_name
+    }
```

Comment:
Sorry for the bringing this up once more, still, for the sake of consistency, I don't think we should introduce a getter for a single field only.

In case you would like to introduce getters for all fields, I think that should be a separate pull request.

Reply:
Removed as suggested. Although I don't agree with the point of maintaining consistency. I think it makes sense to have immutable getters only for a selective set of the fields, with the same rationale behind having different access modifiers for different fields in many languages including rust, but I do agree this might belong to a separate PR, hope this addresses ur concern in this PR, thx!

Reply:
Removed as suggested. Although I don't really agree with the point of maintaining consistency. I think it makes sense to have immutable getters only for a selective set of the fields, with the same rationale behind having different access modifiers for different fields in many languages including rust, but I do agree this might belong to a separate PR, hope this addresses ur concern in this PR, thx!